blocksync/uniqueue.go

```diff
@@ -33,9 +33,6 @@ func (uq *uniQueue) enque(blk *peerBlock) {
 }
 
 func (uq *uniQueue) dequeAll() []*peerBlock {
-	if len(uq.blocks) == 0 {
-		return nil
-	}
```

Comment:
i think it's OK to leave this?

Reply:
It is better to return an empty slice than nil when error type is unavilable in return values

Reply:
ok, then use `return []*peerBlock{}`? 
line 40, 41 is not necessary for the case `if len(uq.blocks) == 0`